### DISCUSSION TITLE
Can't copy grocery list

### DISCUSSION THREAD

**Discussion by Coax88** [mealie-recipes/mealie#2630] [Q&A]
Hi

I'm not able to copy the items in the grocery list, when I use the copy button it says x items are copied to the placeholder, but nothing is copied. Does anyone have a solution?

**Comment by Kuchenpirat** [comment]
Most likely you don't have a secure https connection to your mealie instance configured and "Copy to Clipboard" works only over a https connection which is a limitation in most browsers.

**Reply by Coax88** [reply]
That's correct, I run it on docker on my server. hmm, that's a bummer. Do you know how come it only works in secure browser

**Reply by Kuchenpirat** [reply]
i am pretty sure its an browser / dependency limitation so i don't think that this behaviour will change

**Reply by Coax88** [reply]
sometimes I still think we live in the 80's 😅 we can send spacerockets to Mars but cannot copy in a browser
